helix-term/src/ui/editor.rs

```diff
@@ -171,7 +174,7 @@ impl EditorView {
                 view,
                 view.area,
                 theme,
-                is_focused,
+                is_focused & self.terminal_focused,
```

Comment:
The use of a single `&` here is a little confusing. Wouldn't it have been better to just use `&&` here for clarity?

Reply:
It's already merged, I'm not going to create a PR for this small change.
Anyway, it is not difficult to learn bitwise operators and improve your programming knowledge a little.

Reply:
There's just no reason to add complexity by making this a bit-wise operation rather than a logical AND operation, especially in an open source project with this many moving pieces. I didn't expect a PR be made for this small change, was just pointing it out.

Reply:
A bitwise operation on bool is no less clearer than a logical operation. I prefer the code as is.